qemu/tests/cfg/remote_image_access.cfg

```diff
@@ -79,6 +79,14 @@
             nbd_server_stg1 = ''
             nbd_unix_socket_stg0 = /var/run/nbd_${nbd_image_tag}.sock
             nbd_unix_socket_stg1 = ${nbd_unix_socket_stg0}
+        - repeat_connection_with_unix_socket:
+            type = nbd_unix_connection
+            start_vm = 'no'
+
+            # for socket(type=unix) only
+            nbd_server_stg0 = ''
+            nbd_unix_socket_stg0 = /var/run/nbd_${nbd_image_tag}.sock
+            nbd_connect_cmd = "nbdsh -c 'h.connect_unix("${nbd_unix_socket_stg0}")'"
```

Comment:
nbdsh may not be installed on your testing machine, so
1. you might use a replacement
2. if nbdsh is required, I suggest that set it in cfg to let VT do a pre-check, e.g. 
cmds_installed_host += ' nbdsh'
tc will be ignored if nbdsh is not installed

Reply:
Thanks. Although there maybe other way to test, nbdsh is the easiest way and more close to customer scenario. 
Add "cmds_installed_host += ' nbdsh'" to cfg, as nbdsh depend on package libnbd and python3-libnbd, they won't be installed automatically in current case.